Axiom/sys.lua

```diff
@@ -1,72 +1,43 @@
+local tArgs = {...}
 
-shell.run("clear")
+term.clear()
+term.setCursorPos(1, 1)
 if turtle then
   error("Axiom cannot be run on a turtle, silly.")
 end
---local ip = http.get("http://ipecho.net/plain")
-local ip_processed = nil
-if not ip_processed then
-  ip_processed = "Unable to determine"
-end
-local hasRednet = false
-local rednetDist = 999
-local sides = {"front","back","left","right","top","bottom"}
-local detectedSide = nil
-local modem = nil
-local dangerousStatus = false
-_G.apiErrors = {
-
-}
 
-local dismissed = false
-local remoteChannels = {}
-local remoteSystemInfo = {}
-local speakerSide = nil
-local speakerPresent = false
-local speaker = nil
-local monitor = nil
-local remoteInfo = {
-  systemID = os.getComputerID(),
-  sharingUpdates = true,
-  sharingFiles = true,
-  fileDirectory = "/home/",
-}
-for k,v in ipairs(sides) do
-  if peripheral.isPresent(v) then
-    if peripheral.getType(v) == "speaker" then
-      speakerSide = v
-      speakerPresent = true
-      speaker = peripheral.wrap(v)
-      speaker.playNote("harp",1, 1)
-      break
-    end
-    if peripheral.getType(v) == "monitor" then
-      monitor = peripheral.wrap(v)
-    end
-  end
+--Globals
+_G.apiErrors = {}
+_G.currentUser = "KERNEL"
+_G.productName = "Axiom UI"
+_G.version_sub = "CE"
```

Comment:
The CE bit is no longer used, as this is being used as the main version of Axiom now.